hopp/simulation/technologies/dispatch/power_storage/simple_battery_dispatch_heuristic.py

```diff
@@ -73,22 +73,22 @@ def _set_power_fraction_limits(self, gen: list, grid_limit: list):
     @staticmethod
     def enforce_power_fraction_simple_bounds(power_fraction) -> float:
         """ Enforces simple bounds (0,1) for battery power fractions."""
-        if power_fraction > 1.0:
-            power_fraction = 1.0
-        elif power_fraction < 0.0:
-            power_fraction = 0.0
+        if power_fraction > 0.9:
+            power_fraction = 0.9
+        elif power_fraction < 0:
+            power_fraction = 0
         return power_fraction
 
     def update_soc(self, power_fraction, soc0) -> float:
-        if power_fraction > 0.0:
+        if power_fraction > 0.1:
             discharge_power = power_fraction * self.maximum_power
             soc = soc0 - self.time_duration[0] * (1/(self.discharge_efficiency/100.) * discharge_power) / self.capacity
-        elif power_fraction < 0.0:
+        elif power_fraction < 0.9:
```

Comment:
Doesn't this `if` `elif` statement mean that it will charge only if the `power_fraction` is less than 0.1, is that correct?

Reply:
Yeah this is is kinda misleading. It's equivalent to saying `elif power_fraction < 0.1`, so maybe that's a clearer way to express this